examples/common_patterns_for_connectors/update_and_delete/update_example/README.md

```diff
@@ -0,0 +1,103 @@
+- Updating a single record by specifying the complete composite primary key
+- Updating multiple records by identifying each with its complete primary key
+
+This example is particularly useful for understanding how to properly handle update operations when working with tables that have composite primary keys.
+
+## Requirements
+
+* [Supported Python versions](https://github.com/fivetran/fivetran_connector_sdk/blob/main/README.md#requirements)   
+* Operating system:
+  * Windows: 10 or later (64-bit only)
+  * macOS: 13 (Ventura) or later (Apple Silicon [arm64] or Intel [x86_64])
+  * Linux: Distributions such as Ubuntu 20.04 or later, Debian 10 or later, or Amazon Linux 2 or later (arm64 or x86_64)
+
+## Getting started
+
+Refer to the [Setup Guide](https://fivetran.com/docs/connectors/connector-sdk/setup-guide) to get started.
+
+## Features
+
+- Demonstrates proper handling of update operations with composite primary keys
+- Shows how to connect to and query a PostgreSQL database
+- Illustrates common errors when handling composite primary keys incorrectly
+
+## Configuration file
+
+The connector requires the following configuration parameters to connect to your PostgreSQL database:
+
+```
+{
+  "HOST": "<YOUR_PG_DATABASE_HOST>",
+  "PORT": "<YOUR_PG_DATABASE_PORT>",
+  "DATABASE": "<YOUR_PG_DATABASE_NAME>",
+  "USERNAME": "<YOUR_PG_DATABASE_USERNAME>",
+  "PASSWORD": "<YOUR_PG_DATABASE_PASSWORD>"
+}
+```
+
+Note: Ensure that the `configuration.json` file is not checked into version control to protect sensitive information.
+
+## Requirements file
+
+The connector requires the `psycopg2` Python library to connect to PostgreSQL databases.
+
+```
+psycopg2_binary==2.9.10
+```
+
+Note: The `fivetran_connector_sdk:latest` and `requests:latest` packages are pre-installed in the Fivetran environment. To avoid dependency conflicts, do not declare them in your `requirements.txt`.
+
+## Authentication
+
+This connector uses basic database authentication with a username and password to connect to PostgreSQL. Ensure the database user has appropriate permissions to read from the `product_inventory` table.
+
+
+## Data handling
+
+The connector demonstrates several data operations:
+
+- Data retrieval from PostgreSQL using the `fetch_data` method (lines 48-59)
+- Upsert operations for synchronizing all records (lines 120-123)
+- Update operations with composite primary keys (lines 125-141)
+
+Each record is synchronized to the destination table with its complete set of fields. When updating records, the connector demonstrates how to properly specify all components of a composite primary key.
+
+## Error handling
+
+The connector implements error handling in several areas:
+
+- Database connection errors
+- Data fetching errors
+- Configuration validation
+- General error handling during the update process
+
+The connector also includes warnings about improper update operations that would fail, such as attempting to update with incomplete primary keys.
```

Comment:
```suggestion
The connector also includes warnings about improper update operations that would fail, such as attempts to update records with incomplete primary keys.
```